code/drasil-printers/Language/Drasil/TeX/Print.hs

```diff
@@ -197,7 +197,7 @@ makeTable :: [[Spec]] -> D -> Bool -> D -> D
 makeTable lls r bool t =
   pure (text ("\\begin{" ++ ltab ++ "}" ++ (brace . unwords . anyBig) lls))
   %% pure (text "\\toprule")
-  %% makeRows [head lls]
+  %% makeRows [map (\x -> S "\\textbf{" :+: x :+: S "}") (head lls)]
```

Comment:
You really shouldn't hand-build latex like this, but rather use the facilities in the pretty-printer built for that purpose.